### DISCUSSION TITLE
zh/guide/api/admin/task

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#263] [Announcements]
# zh/guide/api/admin/task

alist中的任务管理可以分为： 接口名 说明 --- --- /upload 上传任务 /copy 复制任务 /aria2_down aria2下载任务 /aria2_transfer aria2转存任务 /qbit_down qbit下载任务 /qbit_transfer qbit转存任务 每种任务都有以下接口： done; undone; del...

https://alist.nn.ci/zh/guide/api/admin/task.html

**Comment by mrgolftech** [comment]
获取已完成任务，返回的data字段为空。

代码如下：
# done task
url9 = "https://alist.example.com/api/admin/task/upload/done"

headers = {
   'Authorization': token
}

res = s.get(url=url9,headers=headers)
print(res.text)

打印信息如下：
{"code":200,"message":"success","data":null}

**Reply by pongfcnkl** [reply]
请问解决了吗？我也是一样不过我返回的是'data': None}

**Reply by mrgolftech** [reply]
返回是正常的，因为我没有创建任务，所以data字段为空

**Comment by dr34m-cn** [comment]
请问下任务完成状态state有哪些可能的值，分别代表什么意思呢？

**Reply by dr34m-cn** [reply]
通过查阅源码，已知到其意义，放在这里方便后来者：https://github.com/xhofe/tache/blob/main/state.go

0-等待中

1-运行中

2-成功

3-取消中

4-已取消

5-出错（将重试）

6-失败（正在执行OnFailed回调）

7-失败（没有剩余重试次数）

8-等待重试

9-任务正在执行OnBeforeRetry回调

**Reply by bf179** [reply]
我就是要找这个！！！
